units/pod_creation_test.go

```diff
@@ -0,0 +1,197 @@
+package units
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/evergreen-ci/cocoa"
+	"github.com/evergreen-ci/cocoa/ecs"
+	cocoaMock "github.com/evergreen-ci/cocoa/mock"
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/cloud"
+	"github.com/evergreen-ci/evergreen/db"
+	evgMock "github.com/evergreen-ci/evergreen/mock"
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/evergreen-ci/evergreen/model/pod"
+	"github.com/evergreen-ci/utility"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCreatePodJob(t *testing.T) {
+	podID := utility.RandomString()
+	p := pod.Pod{
+		ID:     podID,
+		Status: pod.StatusInitializing,
+	}
+
+	j, ok := NewCreatePodJob(&evgMock.Environment{}, &p, utility.RoundPartOfMinute(0).Format(TSFormat)).(*createPodJob)
+	require.True(t, ok)
+
+	assert.NotZero(t, j.ID())
+	assert.Equal(t, podID, j.PodID)
+	assert.Equal(t, pod.StatusInitializing, j.pod.Status)
+}
+func TestCreatePodJob(t *testing.T) {
```

Comment:
nit: add whitespace between L35-36. All functions, variables, constants, structs, etc that are declared at the package level should always have exactly 1 whitespace in between.

Reply:
Got it, the VSCode UI makes it kind of hard to see if I added a newline or not (ie. the suggested `run test/debug test`)